### DISCUSSION TITLE
multiple certificate

### DISCUSSION THREAD

**Discussion by pragyasandeep1998hal** [Keyfactor/ejbca-ce#696] [General]
I want to create multiple certificates, separate certificate for separate NFs. 
tell me how I can do that. Do I need to generate certificate per NF with single End Entity or Multiple End Entity

**Comment by pragyasandeep1998hal** [comment]
Hi can u suggest something.

I want to create multiple certificates, separate certificate for separate NFs.
tell me how I can do that. Do I need to generate certificate per NF with single End Entity or with Multiple End Entity.

I there any reference code u can provide me

**Comment by primetomas** [comment]
You can issue multiple certificates from a single end entity, or you can use multiple end entities. There is many ways you can do the same thing to fit well into your organization. Some use one end entity mapping to a person, server etc, with multiple certificates. Some use a new (random) end entity for each certificate issued.
